### PR TITLE
Removes lies - 99% lag free is replaced by the actual lag percentage

### PR DESCRIPTION
**Pull request by alexkar598**
Lies are bad mkay

:cl:  alexkar598
tweak: 99% lag free replaced by the actual time dilation
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
I like it.

**Comment by ghost** [comment]
New mentor operation, how much lag can we create to make the number go as low as possible

**Review by monster860 — CHANGES_REQUESTED** [review]
This is bad maths, and will result in negative numbers sometimes.

`[round(10000 / (100 + SStime_track.time_dilation_current), 1)]`

^ that one is the one you want

File: yogstation/code/game/world.dm [hunk 1]
```diff
@@ -42,7 +42,7 @@ GLOBAL_LIST_EMPTY(donators)
 	if (server_name)
 		s += "<b>[server_name]</b> &#8212; "
 	
-	s += "<b>[station_name()]] -- 99% LAG FREE</b><br>"; // The station & server name line
+	s += "<b>[station_name()]] -- [100 - round(SStime_track.time_dilation_current,1)]% LAG FREE</b><br>"; // The station & server name line
```
Comment (alexkar598) [inline yogstation/code/game/world.dm]:
```suggestion
	s += "<b>[station_name()]] -- [[round(10000 / (100 + SStime_track.time_dilation_current), 1)]]% LAG FREE</b><br>"; // The station & server name line
```

**Comment by alexkar598** [comment]
Time to commmit based self merge

**Pull request merged**